src/main/resources/META-INF/plugin.xml

```diff
@@ -26,7 +26,7 @@
         >
             <add-to-group group-id="EditorPopupMenu" anchor="first"/>
         </action>
-        <group id="WatermelonContextGroup" popup="true" text="Watermelon Context">
+        <group id="WatermelonContextGroup" popup="true" text="Login to Watermelon">
```

Comment:
Will this group ONLY be for login?

Reply:
I see no other action so yes

Reply:
So when we add the logout action we should rename it?

Reply:
Yes. To something that's more clear than "Watermelon Context" such as "Login/Logout from Watermelon"

Reply:
Should it then not be a group?

Reply:
But we want to have a logout action soon, so why remove it?